src/models/index.ts

```diff
@@ -1,22 +1,40 @@
-import { Sequelize } from 'sequelize';
+import { GroupsModel, GroupsModelStatic } from './groups.model';
+import { UsersModel, UsersModelStatic } from './users.model';
+import { db } from './db';
+import { UsersGroupsModel, UsersGroupsModelStatic } from './users-groups.model';
 
-import { UserModel, UserModelStatic } from './user.model';
+db.sync().then(() => console.log('All models were synchronized successfully.'));
 
-const connectionURI: string =
-    'postgres://agzctnieexodcz:bda6e1e7a30c12426c357809944e754486e23d4ed40c473146051e078c875e15@ec2-54-195-252-243.eu-west-1.compute.amazonaws.com:5432/dfpm81bm9safgr';
+UsersModel.belongsToMany(GroupsModel, {
+    through: UsersGroupsModel,
+    as: 'groups'
+});
 
-const db: Sequelize = new Sequelize(connectionURI, {
-    dialect: 'postgres',
-    dialectOptions: {
-        ssl: true
-    },
-    define: {
-        timestamps: true,
-        paranoid: true
-    },
-    sync: { force: true }
+GroupsModel.belongsToMany(UsersModel, {
+    through: UsersGroupsModel,
```

Comment:
Actually, if delete this linking table and pass here just sting 'UsersGroups' this works for me. As I can see sql for lining table creation is generated and all functionality works well. Can you please double check?